### PR TITLE
Amy dsgvo things

### PR DESCRIPTION
**Pull request by Pondworth**

### CODE REVIEW

File: DSGVO SQL show and delete.sql [hunk 1]
```diff
@@ -0,0 +1,10 @@
+* Auswahl Anzeige eines Kunden per Nachname
+
+SELECT * 
+FROM Kunde
```
Comment (lanetskaya) [inline DSGVO SQL show and delete.sql:4]: SQL is case sensetive. Table is called KUNDE and not Kunde

File: DSGVO SQL show and delete.sql [hunk 2]
```diff
@@ -0,0 +1,10 @@
+* Auswahl Anzeige eines Kunden per Nachname
```
Comment (lanetskaya) [inline DSGVO SQL show and delete.sql:1]: Comments are written with "-- " and not with "*"

File: DSGVO SQL show and delete.sql (hunk 1 above)
[the remark this answers is not in the document]
Comment (Pondworth) [reply DSGVO SQL show and delete.sql:4]: DBeaver autocorrect it always, you have seen that

File: DSGVO SQL show and delete.sql [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+
+SELECT * 
+FROM Kunde 
+WHERE nachname = "Mustermann";
+
+*Löschen eines Kunden per Kundennummer
```
Comment (lanetskaya) [inline DSGVO SQL show and delete.sql:7]: Comment syntax again

File: DSGVO SQL show and delete.sql [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+
+DELETE FROM Kunde
```
Comment (lanetskaya) [inline DSGVO SQL show and delete.sql:9]: KUNDE not Kunde

File: DSGVO SQL show and delete.sql (hunk 1 above)
Comment (Pondworth) [reply DSGVO SQL show and delete.sql:4]: same as before

**Review by lanetskaya — CHANGES_REQUESTED** [review]
Have not tested if the logic works because of sayntax errors

File: DSGVO SQL show and delete.sql (hunk 1 above)
[the remark this answers is not in the document]
Comment (lanetskaya) [reply DSGVO SQL show and delete.sql:4]: So it works for you? DBeaver spits error and refuses to execute for me

[the remark this answers is not in the document]
Comment (Pondworth) [reply DSGVO SQL show and delete.sql:4]: It has, strange, but i can recreate in Editor

**Pull request merged**